tests/feature_resolution_editions.js

```diff
@@ -0,0 +1,263 @@
+/*
+edition = "2023";
```

Comment:
I think we can delete this one now right?

Reply:
I think it's still helpful for me to better understand the precedence of the fields. I've also added inline tests so it's easier for you to read.